### PR TITLE
complete portfolio work section and about section

### PR DESCRIPTION
**Pull request by samiullah997**
i complete the works section and About section in the portfolio project

### CODE REVIEW

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @samiullah997 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## ❌ Invalid Code Review Request

You have submitted a project, that has failed linter checks. Kindly research the font-family linter error and fix it.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -40,6 +42,188 @@ <h4>I’am a Software Developer</h4>
...
+                    the industry's standard</p>
+                <ul class="post-tags-2">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+            <div class="btn-action">
+                <div class="post-btn-enable-2">
+                    <h1 class="btn-text">See Project</h1>
```
Comment (danielufeli) [inline index.html:80]: - [ ] Nice Working on your `Works Section`, kindly make the `see projects`  a link `a` or `button` tag, for better use of HTML semantics.

File: index.html [hunk 2]
```diff
@@ -40,6 +42,188 @@ <h4>I’am a Software Developer</h4>
...
+<!-- Recent Work End -->
+
+<!-- About Section Start -->
+<section class="rectangle">
+    <div class="about">
+        <h1 class="about-title">About me</h1>
+        <p class="about-description">Hello I’m a software developer! I can help you build a product , feature or
+            website Look through some of my work and experience! If you like what you see and have a project you
+            need coded, don’t hestiate to contact me.</p>
+        <div class="about-btn"><span class="about-btn-title">Get My Resume</span></div>
```
Comment (danielufeli) [inline index.html:186]: - [ ] Nice Working on your `About Section`, kindly make the `Get My Resume`  a link `a` or `button` tag, for better use of HTML semantics.

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @samiullah997 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @samiullah997 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**